static/js/components/Group.jsx

```diff
@@ -335,17 +335,20 @@ const Group = () => {
             <List component="nav" className={classes.padding_bottom}>
               {group.streams?.map((stream) => (
                 <div key={stream.name}>
-                  <ListItem>
+                  <ListItem key={stream.name}>
                     <ListItemText primary={stream.name} />
                   </ListItem>
                   <List component="nav" disablePadding>
                     {group.filters?.map((filter) =>
                       filter.stream_id === stream.id ? (
-                        <ListItem button component="a" key={filter.id} href="#">
-                          <ListItemText
-                            className={classes.nested}
-                            primary={filter.name}
-                          />
+                        <ListItem button key={filter.id}>
+                          <Link to={`/filter/${filter.id}`}>
```

Comment:
```suggestion
                          <Link to={`/filter/${filter.id}`} className={classes.filterLink}>
```
And define filterLink above to have `width: "100%"`. Right now, the link only exists on the text of the filter name and not along the whole row in the list, but the whole row is highlighted on hover, leading to non-intuitive behavior.

Reply:
Nice catch, thanks! Fixed